webpack.config.js

```diff
@@ -48,7 +48,6 @@ const publicJsConfig = {
     CounterScript: './assets/src/blocks/Counter/CounterScript.js',
     SplitTwoColumnsScript: './assets/src/blocks/SplitTwoColumns/SplitTwoColumnsScript.js',
     ArticlesScript: './assets/src/blocks/Articles/ArticlesScript.js',
-    CookiesScript: './assets/src/blocks/Cookies/CookiesScript.js',
```

Comment:
Is the Cookie scripts removal part of this PR ?

Reply:
Oh no! Good catch 🙏